### ISSUE TITLE
How do i make my eventhandlers working while using method two?

### ISSUE THREAD

**Issue by shamun** [onsip/SIP.js#159]
How to make this Method 2 eventHandlers get work?

```
var eventHandlers = {
  'progress': function(e) {
    console.info('Progress: ', e);
  },
  'failed': function(e) {
    console.info('Failed: ', e);
  },
  'terminated':function(e) {
    console.info('Terminated: ', e);
  },
  'accepted': function(e){
    console.info('Accepted: ', e);
  },
  'rejected': function(e){
    console.info('Rejected: ', e);
  },
  'cancel': function(e) {
    console.info('Cancel: ', e);
  }
};

var options = {
  'eventHandlers':    eventHandlers,
  media: {
    constraints: {
      audio: true,
      video: false
    },
    render: {
      remote: {
          video: document.getElementById('remoteView')
      },
      local: {
          video: document.getElementById('selfView')
      }
    }
  }
};

function make_call() {
  try {
    rtcSession = MyPhone.invite(dial_to, options); //9664
  } catch(e){
    console.info('make_call(): ', e);   
    return;
  }  
}

// Method 1: works only such as rtcSession.on('...', ...);
make_call(); // Method 2: all the eventHandlers does not work 

```

**Comment by egreenmachine** [comment]
Please see our [documentation for invite](http://sipjs.com/api/0.6.0/ua/#invitetarget-options). Event handlers is not a valid option for our invite API and I would not expect event handlers to work in that case. Since this is not an issue with our library I am going to close this.

Additionally I have noticed that the issues you are opening are mostly questions on how to use the library instead of actual problems with the library. Please use our [mailing list](https://groups.google.com/forum/#!forum/sip_js) for these types of questions.

**Comment by shamun** [comment]
OK - no problem. 

Its confusing for people coming from JsSIP. Because everywhere SIP.js is known as JsSIP replacement.

Everywhere People keep saying SIP.js is JsSIP but its not true cause you already mentioned library itself does not allow it.

**Comment by wakamoleguy** [comment]
> Everywhere People keep saying SIP.js is JsSIP but its not true

I couldn't agree more.  It has been over a year since we forked from JsSIP.  While the names may be similar, many of the features, functionality, and attitudes are really quite different.  I made a few comments on our mailing list recently about [the differences between SIP.js and JsSIP](https://groups.google.com/forum/#!topic/sip_js/aFqTAyQz8i4).  It may be worth a read, if you are curious.
